src/modules/Permissions/dialogs/ManagePermissions/ManagePermissions.tsx

```diff
@@ -116,6 +116,14 @@ export const ManagePermissionsModal: React.FC<
   const [removeAcls, setRemoveAcls] = useState<EnhancedAclBinding[]>([]);
   const [isOpenPreCancelModal, setIsOpenPreCancelModal] =
     useState<boolean>(false);
+  const [
+    isExpandedExistingPermissionSection,
+    setIsExpandedExistingPermissionSection,
+  ] = useState<boolean>(false);
+  const [
+    isExpandedAssignPermissionsSection,
+    setIsExpandedAssignPermissionsSection,
+  ] = useState<boolean>(false);
```

Comment:
I'm not sure if these values represent the default state, but when these contents first load in modal, you should see:

- **Review existing permissions** is **collapsed**
- **Assign permissions** is **expanded**

Reply:
@jgiardino  yes, make sense. I'll update default state for assign permission. Thanks for the feedback.